webview/src/getStarted/components/NoData.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import pyTorch from '../snippets/pyTorch.py'
+import huggingFace from '../snippets/huggingFace.py'
+import keras from '../snippets/keras.py'
+import pythonApi from '../snippets/pythonApi.py'
+import { CodeBlock } from '../../shared/components/codeBlock/CodeBlock'
+import { CodeSlider } from '../../shared/components/codeSlider/CodeSlider'
+import { EmptyState } from '../../shared/components/emptyState/EmptyState'
+
+export const NoData: React.FC = () => {
+  return (
+    <EmptyState>
+      <div>
+        <h1>Your project contains no data</h1>
+        <div>
```

Comment:
For accessibility purposes, we should probably wrap the text in a `p` tag.

Reply:
I know, it's only because of `<CodeBlock language="python" inline>save_dvc_exp=True</CodeBlock>` which uses `<div>` inside.

Reply:
The `div` on line 27 of `CodeBlock.tsx`? Why not replace that with a `span` and apply any needed styles to it with CSS?

Reply:
Because styling is done by the lib.